src/app/kuma/components/ApplicationShell.vue

```diff
@@ -80,6 +80,13 @@
                   Feedback
                 </a>
               </KDropdownItem>
+
+              <KDropdownItem
+                :item="{
+                  to: { name: 'onboarding-welcome' },
+                  label: 'Onboarding',
+                }"
+              />
```

Comment:
There should be a conditional or something here so that in Mink you can't do onboarding still no?

Reply:
This ApplicationShell doesn't happen in MinK, theres another implementation in MinK